src/main.rs

```diff
@@ -1,14 +1,46 @@
-use std::path::PathBuf;
+use std::{io::Write, path::PathBuf};
+
+use clap::Parser;
 
 use jakt::{Compiler, JaktError, Span};
 
+#[derive(Parser, Debug)]
+struct CLIArgs {
+    /// Binary directory
+    #[clap(long = "binary-directory", short = 'o')]
```

Comment:
No spaces around key-value pairs in attribute macros, usually

Reply:
rustfmt complained in CI when I removed the spaces

Reply:
ah strange, seems like I don't write enough of these :)